src/Fantomas/FormatConfig.fs

```diff
@@ -57,6 +58,7 @@ type FormatConfig =
           MaxArrayOrListWidth = 40
           MaxLetBindingWidth = 40
           MultilineBlockBracketsOnSameColumn = false
+          KeepIfThenInSameLine = false
```

Comment:
Please also mention the new setting in the [documentation ](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#--config-path-to-file-or-folder) and [schema.json](https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/schema.json).

No need to write proper documentation. I'll tackle this #725 .
Just make sure the setting is mentioned there so I won't forget.